src/SortingAlgorithms.jl

```diff
@@ -619,4 +640,54 @@ function sort!(v::AbstractVector, lo::Int, hi::Int, ::CombSortAlg, o::Ordering)
     return sort!(v, lo, hi, InsertionSort, o)
 end
 
+function sort!(v::AbstractVector, lo::Int, hi::Int, ::BitonicSortAlg, o::Ordering)
+    return bitonicsort!(view(v, lo:hi), o::Ordering)
+end
+
+function bitonicsort!(data, o::Ordering)
+    N = length(data)
+    for n in 1:ceil(Int, max(0, log2(N)))
+        bitonicfirststage!(data, Val(n), o::Ordering)
```

Comment:
Can we simply force N and n to be integers somehow? And what makes it unstable in the first place?

The idea is indeed to be forcing compilation with knowledge of the input size, because this seems necessary to trigger compiler optimizations. I saw a lot more vectorization in the machine code with that, and I believe it won't be really interesting unless we do this.

We might have a limit above which we don't use compile-time-known values, if that's an issue.